packages/components/src/radio-control/index.tsx

```diff
@@ -93,6 +65,7 @@ export function RadioControl(
 						aria-describedby={
 							!! help ? `${ id }__help` : undefined
 						}
+						ref={ forwardedRef }
```

Comment:
Good point — I had actually missed that `map` statement while making these quick changes.

Turns out that is is more complicated than expected — we could forward the `ref` to `BaseControl`, but `BaseControl` currently doesn't accept refs.

I tried to quickly add ref forwarding to `BaseControl`, but I ran into type issues — mainly to do with the fact that `VisualLabel` is currently defined as a static property on `BaseControl` (e.g. `BaseControl.VisualLabel`), and adding ref forwarding causes an error with that

![Screenshot 2022-06-10 at 09 19 08](https://user-images.githubusercontent.com/1083581/173011835-85ba2a33-6655-4ca2-bf42-254ab4acde05.png)

Since this is not currently a priority, I've decided to stop investigating this further for now. What do folks think is the best way forward? I'm also happy to drop this entirely for the time being.

Reply:
I agree we can drop it for now, unless we have a concrete use case. I came across [this note in the React docs](https://reactjs.org/docs/forwarding-refs.html#note-for-component-library-maintainers) that forwarding refs is a breaking change, and while I'm not 100% sure about what that entails, it did give me pause.

Reply:
Makes sense. Will close this PR for now